DPG/Tasks/AOTTrack/PID/qaPIDTPC.cxx

```diff
@@ -50,6 +50,9 @@ struct tpcPidQa {
   static constexpr std::string_view hnsigma_pt[Np] = {"nsigma/pt/El", "nsigma/pt/Mu", "nsigma/pt/Pi",
                                                       "nsigma/pt/Ka", "nsigma/pt/Pr", "nsigma/pt/De",
                                                       "nsigma/pt/Tr", "nsigma/pt/He", "nsigma/pt/Al"};
+  static constexpr std::string_view hnsigma_eta[Np] = {"nsigma/eta/El", "nsigma/eta/Mu", "nsigma/eta/Pi",
```

Comment:
hnsigma_eta -> hnsigma_p_eta_Ncl
nsigma/eta (in path) --> nsigma/sparsePinEtaNcl